debian/tests/common-functions

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+
+check_arch() {
+    local architecture
+    architecture=$(dpkg --print-architecture)
```

Comment:
Not strictly needed here but I would still suggest to use quotes around all "$" calls just for good measure.